tests/test_playbooks/setting_info.yml

```diff
@@ -7,6 +7,8 @@
     - vars/server.yml
   tasks:
     - include_tasks: tasks/setting.yml
+      vars:
+        setting_value: ''
```

Comment:
I think it's confusing that we set the setting name in [tasks/setting.yml](https://github.com/theforeman/foreman-ansible-modules/blob/85698bc686e4a8301a889c54a976788423459774/tests/test_playbooks/tasks/setting.yml#L4) but the value here. Is that just me?

Reply:
No, but I guess you're right and we could be explicit here too.